### PR TITLE
add: keep email in forgot password

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Change links in forgot password and login to pass the email in state
- Change initial values in the email field to put the email if has in state.

### CODE REVIEW

File: src/components/ForgotPassword/ForgotPassword.js [hunk 1]
```diff
@@ -56,6 +63,23 @@ const ForgotPassword = ({ intl, dependencies: { dopplerLegacyClient } }) => {
...
+        to={{
+          pathname: '/login',
+          state: { email: email },
+        }}
+        className="forgot-link"
+      >
+        <span className="triangle-right" />
+        {sentTimes > 0
+          ? _('forgot_password.back_login_after_forgot')
+          : _('forgot_password.back_login')}
```
Comment (andresmoschini) [inline src/components/ForgotPassword/ForgotPassword.js]: ❤️

**Comment by andresmoschini** [comment]
Nice!

**Comment by GusBaamonde** [comment]
👍

**Comment by fcoronelmakingsense** [comment]
> 👍 To try! https://cdn.fromdoppler.com/doppler-webapp/int-build683/#/login

Thanks @GusBaamonde

**Comment by fcoronelmakingsense** [comment]
Update Pr fixing send forgot password form

File: src/components/ForgotPassword/ForgotPassword.js [hunk 2]
```diff
@@ -14,29 +14,36 @@ import { InjectAppServices } from '../../services/pure-di';
...
  *
  * @param { Object } props
  * @param { import('react-intl').InjectedIntl } props.intl
  * @param { import('../../services/pure-di').AppServices } props.dependencies
  */
-const ForgotPassword = ({ intl, dependencies: { dopplerLegacyClient } }) => {
+const ForgotPassword = ({ intl, location, dependencies: { dopplerLegacyClient } }) => {
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
-  const [sentTimes, setSentTimes] = useState(0);
+  const [sentTimes, setSentTimes] = useState(null);
```
Comment (andresmoschini) [inline src/components/ForgotPassword/ForgotPassword.js]: Could you rename it?

**Comment by andresmoschini** [comment]
Please, rename _sentTimes_ with a descriptive name, test it and merge.

File: src/components/ForgotPassword/ForgotPassword.js (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/components/ForgotPassword/ForgotPassword.js]: changed!

File: src/components/ForgotPassword/ForgotPassword.js [hunk 3]
```diff
@@ -14,29 +14,36 @@ import { InjectAppServices } from '../../services/pure-di';
...
  *
  * @param { Object } props
  * @param { import('react-intl').InjectedIntl } props.intl
  * @param { import('../../services/pure-di').AppServices } props.dependencies
  */
-const ForgotPassword = ({ intl, dependencies: { dopplerLegacyClient } }) => {
+const ForgotPassword = ({ intl, location, dependencies: { dopplerLegacyClient } }) => {
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
-  const [sentTimes, setSentTimes] = useState(0);
+  const [sentEmail, setSentEmail] = useState(null);
```
Comment (andresmoschini) [inline src/components/ForgotPassword/ForgotPassword.js]: ❤️

**Comment by andresmoschini** [comment]
Nice! if you already tested it fine, please :shipit:

**Pull request merged**